src/plugins/intel_cpu/tests/functional/custom/subgraph_tests/src/common/concat_multiple_query_sdp.cpp

```diff
@@ -72,7 +74,8 @@ class ConcatMultiQuerySDPTest : public testing::WithParamInterface<ConcatMultiQu
             result << ")_";
         }
         result << "Prc=" << qkvType << "_";
-        result << "HasShapeOf=" << hasShapeof << "_";
+        result << "ForceKVU8" << forceKVU8 << "_";
```

Comment:
How about using `ElementType kvCacheType;` for better understanding?

Reply:
> How about using `ElementType kvCacheType;` for better understanding?

accept it.